chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -322,6 +322,9 @@ def find_node_boot_mac_addresses(node, admin_data_net)
             cloud_available = true if name.include? "Cloud"
           end
 
+          ntp_servers = search(:node, "roles:ntp-server")
```

Comment:
we don't seem to be very consistent on this. there is no other usage of search_env_filtered in crowbar-core (actually I can find it only in the nova barclamp, which is strange..)